examples/coaptest/coaptest.ino

```diff
@@ -21,7 +21,7 @@ void callback_response(CoapPacket &packet, IPAddress ip, int port) {
 
   char p[packet.payloadlen + 1];
   memcpy(p, packet.payload, packet.payloadlen);
-  p[packet.payloadlen] = NULL;
+  p[packet.payloadlen] = "\0";
```

Comment:
The original code uses NULL, which works but is semantically incorrect for a char.
Replacing it with "\0" is also incorrect, as "\0" is a string literal (char *).
The correct fix would be:
`p[packet.payloadlen] = '\0';`